src/user_crate/state_generated.rs

```diff
@@ -126,8 +126,7 @@ impl StateGenerated {
 
     #[tracing::instrument(level = "debug", skip_all, fields(fn_oid = %self.fn_oid))]
     pub(crate) fn lib_rs(&self) -> eyre::Result<syn::File> {
-        let mut skeleton: syn::File =
-            syn::parse_str(include_str!("./skeleton.rs")).wrap_err("Parsing skeleton code")?;
+        let mut skeleton: syn::File = syn::parse_str("use ::pgx::*;").wrap_err("Parsing skeleton code")?;
```

Comment:
I think this won't fail so you can do this instead I think:

```suggestion
        let mut skeleton: syn::File = syn::parse_quote!(use ::pgx::*;");
```